plugins/router/freebox

```diff
@@ -0,0 +1,316 @@
+#!/bin/bash
+# -*- sh -*-
+
+: << =cut
+
+=head1 NAME
+
+freebox - Plugin to monitor stats of a Freebox
+
+The ideal usage is also used as an updated warning limit.
```

Comment:
Could you rephrase this, please?
(I do not understand it spontaneously)

Reply:
Ah, this was from a different plugin I took as a base to work on this one. It's not relevant here.